app/src/main/java/ch/epfl/culturequest/social/PictureAdapter.java

```diff
@@ -165,9 +173,40 @@ public PictureViewHolder(@NonNull View itemView) {
             profilePicture = itemView.findViewById(R.id.profile_picture);
             like = itemView.findViewById(R.id.like_button);
             delete = itemView.findViewById(R.id.delete_button);
-
         }
 
     }
+
+    private Snackbar createSnackbar(View v) {
```

Comment:
I think this method should be in the utils folder has this may be used in other parts of the app

Reply:
Done by Thomas ✔️